.gitignore

```diff
@@ -8,3 +8,9 @@ __pycache__
 *.csv
 *.json
 .DS_Store
+terraform.tfvars
+terraform.tfstate
+terraform.tfstate.backup
+.terraform/providers/registry.terraform.io/hashicorp/aws/4.67.0/darwin_arm64/terraform-provider-aws_v4.67.0_x5
```

Comment:
you can change some of this to *.tfvars , *.tfstate -- I think this should cover the back up but may have to ask Remi